addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -118,29 +118,6 @@ registerModel({
             }
             return insertAndReplace(messageViewsData);
         },
-        /**
-         * @private
-         * @returns {string[]}
-         */
-        _computeTextInputSendShortcuts() {
-            if (!this.thread) {
-                return;
-            }
-            if (!this.messaging.device) {
-                return;
-            }
```

Comment:
This guard was not kept, is it on purpose?

Reply:
Good catch, it's not on purpose.
It's safer to keep current behaviour of no send shortcut in mobile, so that the `RETURN`/`ENTER` on soft-keyboard can actually be used to make a line return.